xcode/Subconscious/Shared/Components/Common/Text/TranscludeBlockLayoutFragment.swift

```diff
@@ -0,0 +1,155 @@
+//
```

Comment:
Nit: should we create a `Subconscious/Shared/Components/Common/Transclude/` directory and place this and other transclude code in it?